scss/components/_search-bar.scss

```diff
@@ -114,13 +116,13 @@
         color: $white;
         border: none;
         padding: 0;
-        background-color: $primary;
+        background-color: #7e9a3a;
         height: ($baseline * 6);
         position: relative;
 
         &:hover,
         &:focus {
-            background-color: darken($primary, 8%);
+            background-color: #738c35;
```

Comment:
When using colours we try and keep them in the [_colours SCSS file](https://github.com/ONSdigital/sixteens/blob/develop/scss/utilities/_colours.scss).
I think #738c35 is close to 'Sycamore' and  #7e9a3a is close to 'Sushi' but could be wrong about the names, if they aren't named might be worth us naming them in the colours file something appropriate and referencing them here.